src/api/v1/lobby.rs

```diff
@@ -159,41 +167,58 @@ async fn lobby_try_contribute_test() {
         session_id.clone(),
         Extension(shared_state.clone()),
         Extension(db.clone()),
-        Extension(transcript.clone())
-    ).await;
-    assert!(matches!(unknown_session_response, TryContributeResponse::UnknownSessionId));
+        Extension(transcript.clone()),
+    )
+    .await;
+    assert!(matches!(
```

Comment:
// Definitely not blocking this PR since you haven't touched it at all

Since we are making lots of quality of life changes, this is not ideal. When the test fails, logs only mention the expected value (no actual value).

There is an `assert_matches!` macro in unstable Rust that might work better. I wasn't sure what would be the best way to bring that macro here.

Reply:
Oh, that's cool. I'm okay with an unstable dependency in test/dev, but like to avoid it in prod code.